webview/src/setup/components/remotes/AmazonS3.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
[F] I have split these components right out to a very granular level.

[Q] Have I gone too far?

Reply:
I would have gone further as `AmazonS3` and `AzureBlobStorage` share a wrapper `<div className={styles.storageDetails}>` and a `<CloudVersioning>` block, so I don't think you went too far. It's fine as is though.